store/src/java/com/zimbra/cs/service/account/GetInfo.java

```diff
@@ -461,4 +475,26 @@ protected Element encodeChildAccount(Element parent, Account child, boolean isVi
     private void doDiscoverRights(Element eRights, Account account, Set<Right> rights) throws ServiceException {
         DiscoverRights.discoverRights(account, rights, eRights, false);
     }
+
+    private boolean checkIfPowerpasteInstalled() {
+        String libLocation = "/opt/zimbra/common/lib/pasteitcleaned";
+        String extLoc = "/opt/zimbra/lib/ext/powerpaste-ext/zm-powerpaste-extension.jar";
+        try {
+            File lib = new File(libLocation);
+            File ext = new File(extLoc);
+            if (lib.exists() && lib.isDirectory() && lib.canRead()) {
+                if (ext.exists() && ext.isFile() && ext.canRead()) {
```

Comment:
Can we do this check by checking the handler registered like its done in ExtensionDispatcherServlet?

Reply:
That can not be done here. and also until we  restart mailbox the extension would still present in the list. so I think it is better to check in that way.